bin/varnishd/cache/cache_backend.c

```diff
@@ -143,6 +143,14 @@ vbe_connwait_dequeue_locked(struct backend *bp, struct backend_cw *cw)
 	cw->cw_state = CW_DEQUEUED;
 }
 
+static void
+backend_cw_fini(struct backend_cw *cw)
+{
+	CHECK_OBJ_NOTNULL(cw, BACKEND_CW_MAGIC);
+	AN(cw->cw_state != CW_QUEUED);
```

Comment:
Use the `assert()` macro if you pass an actual condition.

Reply:
Fixed.